.github/workflows/infrastructure.yml

```diff
@@ -23,150 +23,418 @@
 #
 # TO-DO
 # *****
-# 0. Add --internal-only true \ to CAE
 # 1. Add delete locks
 # 2. Add log workspace
 # 3. Add Insight
 # 4. Add monitor
-# 5. Add API management service
-# 6. Add Key vault
-# 7. Add VNET peer (x-subscription RBAC SP)
+# 5. Add Key vault
 
 name: Infrastructure 🔨
```

Comment:
This is quite a long file, is it possible to split it up at all to make it more readable?

Reply:
Agreed. But it will change in the future. Each Microservice will have its own infrastructure and deployments. So the file will get smaller. 
I think that can take place as part of a different PR

Reply:
That's fine with me 👍